### DISCUSSION TITLE
TeslaMate for Rivian?

### DISCUSSION THREAD

**Discussion by Ken-Yn** [teslamate-org/teslamate#3354] [Ideas]
Are there any plans to create a version of TeslaMate for Rivian vehicles?  I know TeslaFi has a beta version for Rivian.

**Comment by ZakWinnick** [comment]
Just saw this in a search...There are more of us out there now that would love to have something like TeslaMate for our trucks.

**Comment by nerdknob** [comment]
Chiming in that I’d love a Rivian version as well!

**Comment by JustinJS** [comment]
I would also love it, right now I am using Electrafi, but I would prefer to self host. I'd gladly throw the devs a few bucks.

**Comment by dealthmalfoy** [comment]
I will like a self hosted platform for my Rivian as well

**Comment by agrieco** [comment]
Did anyone get anywhere on this?

**Comment by mwkrieger** [comment]
Add me to the list! Would love a version for Rivian.

**Comment by sbenbur** [comment]
Following
